build_tools/style.fish

```diff
@@ -44,7 +44,7 @@ if test $all = yes
         end
     end
     set fish_files $workspace_root/{benchmarks,build_tools,etc,share}/**.fish
-    set python_files {doc_src,share,tests}/**.py
+    set python_files .
```

Comment:
Yeah given that ruff tries to respect gitignore, I'm good with this change.
(prior discussion for shellcheck is at https://github.com/fish-shell/fish-shell/pull/11608)